MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/location/LocationModesActivity.java

```diff
@@ -233,8 +233,10 @@ private void toggleMapStyle() {
       return;
     }
 
-    String styleUrl = Style.DARK.equals(mapboxMap.getStyle().getUri()) ? Style.LIGHT : Style.DARK;
-    mapboxMap.setStyle(new Style.Builder().fromUri(styleUrl));
+    mapboxMap.getStyle(style -> {
```

Comment:
NIT: testing leftover?

Reply:
It's a general improvement to the test app. It's a better practice to use async `getStyle() ` callback.

Reply:
Ah, the `style` is for `getUri`. Didn't spot at first :+1: